q2_diversity_lib/tests/test_alpha.py

```diff
@@ -15,23 +15,28 @@
 from q2_types.feature_table import BIOMV210Format
 from q2_types.tree import NewickFormat
 from q2_diversity_lib import (faith_pd, pielou_evenness, observed_features,
-                              shannon_entropy)
+                              shannon_entropy, alpha, translate_metric_name)
+from qiime2 import Artifact
 
-nonphylogenetic_measures = [observed_features, pielou_evenness,
-                            shannon_entropy]
+nonphylogenetic_metrics = alpha.METRICS['NONPHYLO']['IMPL']
+metric_name_translations = alpha.METRICS['METRIC_NAME_TRANSLATIONS']
```

Comment:
These are only used one time - I don't think it is worth polluting the namespace with these new variables. Please remove these.

`nonphylogenetic_metrics` is only 4 chars less than `METRICS['NONPHYLO']['IMPL']`, and if you drop the extra `METRIC_` in `METRIC_NAME_TRANSLATIONS` you'll also only be saving yourself 4 chars, not worth it IMO to mostly just add an extra layer of indirection.